<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lecture des fichiers .dat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Champs E(t)</string>
      </property>
      <layout class="QHBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>Open Ek.dat file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save in .xcls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colonne1">
         <property name="text">
          <string>temps(fs)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colonne2">
         <property name="text">
          <string>Intensité</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colonne3">
         <property name="text">
          <string>Phase(rad)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colonne4">
         <property name="text">
          <string>Partie reel(E(t))</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colonne5">
         <property name="text">
          <string>Partie Img(E(t))</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="spectreGroupBox">
      <property name="title">
       <string>Spectre</string>
      </property>
      <layout class="QHBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="openSpeckButton">
         <property name="text">
          <string>Open Speck.dat file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveSpeckButton">
         <property name="text">
          <string>Save Spectre.xls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="longueurOnde">
         <property name="text">
          <string>longueur d'onde</string>
      </property>
     </widget>
    </item>
       <item>
        <widget class="QCheckBox" name="intensiteSpectrale">
         <property name="text">
          <string>intensité spectrale</string>
   </property>
  </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="phaseSpectrale">
         <property name="text">
          <string>Phase Spectrale(rad)</string>
         </property>
 </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mplWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
